target/linux/ramips/image/mt7620.mk

```diff
@@ -924,14 +915,16 @@ define Device/ralink_mt7620a-v22sg-evb
 endef
 TARGET_DEVICES += ralink_mt7620a-v22sg-evb
 
-define Device/ravpower_wd03
```

Comment:
You should touch this original recipe only when you touch the original .dts!

Reply:
Sorry, not quite understanding this one 😞. It seemed like folks wanted it changed / corrected, to match the vendor name better?

Reply:
@xabolcs can you rephrase what you mean? This commit changes both recipe and DTS at same time, unless you mean something else...

Reply:
@mpratt14 I meant exactly that.

Somehow I saw that the HooToo dts was renamed to dtsi.

Reply:
Yeah this is a feature of git where if you have a deletion and a new file but they are more than 50% similar it counts it as a renamed file instead

I mentioned earlier and in forum that this makes it hard to review and understand what this commit does in the future. Solution would be to do all the renaming in a separate commit

Reply:
It's OK to me to see one of the dts as a rename, but I missed the other here.
I was looking this PR through app.